src/Router/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './types';
```

Comment:
i would like suggest to avoid index.ts files with only exports from another file. it increase count of links to this files, and increase coupling. im not sure about helpful of this type of file

Reply:
I used to clean imports. 
From: 
`
import { FC } from 'react';
import { Routes, Route, BrowserRouter } from 'react-router-dom';
import { RouterProvider } from "../../Router/RouterContext";
import { RouteName } from "../../Router/types";
import { Library } from "../../pages/library/library";
import { Login } from "../../pages/auth/login/login";
import { Registration } from "../../pages/auth/registration/registration";
import { Video } from "../../pages/library/video/video";
`
to
`
import { FC } from 'react';
import { Routes, Route, BrowserRouter } from 'react-router-dom';
import { Library, Login, Registration, Video } from '../../pages';
import { RouteName, RouterProvider } from '../../Router';
`

Reply:
Yes, i understand. But first approach is more proffered. Code is cleaner, but code is much more coupled